src/Input/Factory/FieldFactory.php

```diff
@@ -45,7 +46,9 @@ public function create(
             }
 
             $type = $schema->type;
-            if (isset($schema->allOf)) {
+            if (isset($schema->oneOf)) {
+                throw new InvalidSpecificationException('oneOf keyword is not currently supported.');
```

Comment:
Ye, sure. Actually, a lot of things are missing in README. I agree that a good README file is one of the most important things for an open-source package.
It can come anytime though, after the version release. I have a plan to write it properly, just I'll be on vacation starting from tomorrow, so I'm lack of time now:(